monolith/views/stories.py

```diff
@@ -10,8 +10,16 @@
 @stories.route('/stories')
 def _stories(message=''):
     allstories = db.session.query(Story)
-    return render_template("stories.html", message=message, stories=allstories, like_it_url="http://127.0.0.1:5000/stories/like/")
+    return render_template("stories.html", message=message, stories=allstories)
```

Comment:
In this line you should add the variable like_it_url, because is read by the html but not defined in this method.
`return render_template("stories.html", message=message, stories=allstories, like_it_url="/stories/like/")`

Reply:
I wanted to remove the possibility of liking (/disliking) a post from the stories list.
If you agree with that I'll instead remove instead the variable from the HTML page.

Reply:
Sure! Maybe we could also let the user see a preview of the story, limiting the number of character to be printed with relative url that points to the /story/<id>.

Reply:
Liked to https://github.com/sduoase/STAR_DICES/issues/24. I'll do it, that's my job!